package.json

```diff
@@ -12,22 +12,16 @@
     "build": "babel -d dist src",
     "check-coverage": "nyc check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
     "lint": "eslint .",
-
```

Comment:
Don't worry about this. `npm` reformats the `package.json` anytime it changes it. You're good :+1: